Undo test changes expecting SameSite=Lax cookies by default

charliewolfe opened the pull request:
#### 9d5f18152a927cd0136adca280385f8e64ebb532
<pre>
Undo test changes expecting SameSite=Lax cookies by default
<a href="https://bugs.webkit.org/show_bug.cgi?id=281984">https://bugs.webkit.org/show_bug.cgi?id=281984</a>
<a href="https://rdar.apple.com/138487628">rdar://138487628</a>

Reviewed by Matthew Finkel.

* LayoutTests/http/tests/cookies/resources/testharness-helpers.js:
* LayoutTests/http/tests/cookies/same-site/fetch-after-navigating-iframe-in-cross-origin-page-expected.txt:
* LayoutTests/http/tests/cookies/same-site/fetch-after-navigating-iframe-in-cross-origin-page.html:
* LayoutTests/http/tests/cookies/same-site/fetch-in-cross-origin-service-worker-expected.txt:
* LayoutTests/http/tests/cookies/same-site/fetch-in-cross-origin-service-worker.html:
* LayoutTests/http/tests/cookies/same-site/popup-cross-site-post.html:
* LayoutTests/http/tests/cookies/same-site/resources/fetch-after-navigating-iframe-in-cross-origin-page.py:
* LayoutTests/http/tests/cookies/same-site/resources/fetch-in-cross-origin-service-worker.html:
* LayoutTests/imported/w3c/web-platform-tests/html/anonymous-iframe/cookie.tentative.https.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/cross-origin-embedder-policy/anonymous-iframe/cookie.tentative.https.window-expected.txt:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios-17/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/285696@main">https://commits.webkit.org/285696@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/49d4879cedbea2f78476d5aaa1bae00c282988ad

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/73535 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/52964 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/26343 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/77824 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/24773 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/62095 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/749 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/57800 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/16214 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/76602 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/47894 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/63270 "Passed tests") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/38211 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/44503 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/20756 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/23105 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/66283 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/21105 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/79422 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/851 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/332 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/66181 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/992 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/63282 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/65460 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/9317 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/7492 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/11329 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/814 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/844 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/830 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/850 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

sysrqb commented on LayoutTests/imported/w3c/web-platform-tests/html/anonymous-iframe/cookie.tentative.https.window-expected.txt:
```diff
@@ -1,13 +1,13 @@
 PASS Setup
 FAIL Anonymous same-origin iframe is loaded without credentials assert_equals: expected (undefined) undefined but got (string) "same_origin"
-PASS Anonymous cross-origin iframe is loaded without credentials
+FAIL Anonymous cross-origin iframe is loaded without credentials assert_equals: expected (undefined) undefined but got (string) "cross_origin"
```
These WPT tests are expecting SameSite=Lax?

charliewolfe replied:
Yes

charliewolfe committed:
Undo test changes expecting SameSite=Lax cookies by default
https://bugs.webkit.org/show_bug.cgi?id=281984
rdar://138487628

Reviewed by Matthew Finkel.

* LayoutTests/http/tests/cookies/resources/testharness-helpers.js:
* LayoutTests/http/tests/cookies/same-site/fetch-after-navigating-iframe-in-cross-origin-page-expected.txt:
* LayoutTests/http/tests/cookies/same-site/fetch-after-navigating-iframe-in-cross-origin-page.html:
* LayoutTests/http/tests/cookies/same-site/fetch-in-cross-origin-service-worker-expected.txt:
* LayoutTests/http/tests/cookies/same-site/fetch-in-cross-origin-service-worker.html:
* LayoutTests/http/tests/cookies/same-site/popup-cross-site-post.html:
* LayoutTests/http/tests/cookies/same-site/resources/fetch-after-navigating-iframe-in-cross-origin-page.py:
* LayoutTests/http/tests/cookies/same-site/resources/fetch-in-cross-origin-service-worker.html:
* LayoutTests/imported/w3c/web-platform-tests/html/anonymous-iframe/cookie.tentative.https.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/cross-origin-embedder-policy/anonymous-iframe/cookie.tentative.https.window-expected.txt:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios-17/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/285696@main
